packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -368,4 +395,26 @@ export class ApiConnectorImpl {
     }
     return node;
   }
+
+  public buildAPIKeyAuthQuestion(): QTreeNode {
+    const node = new QTreeNode({
+      name: Constants.questionKey.apiAPIKeyLocation,
+      type: "singleSelect",
+      staticOptions: [headerLocationOption, queryLocationOption],
+      title: getLocalizedString("plugins.apiConnector.getQuestion.apiKeyLocation.title"),
+    });
+    node.condition = { equals: APIKeyAuthOption.id };
+
+    const headerKeyNameQuestionNode = new QTreeNode(buildAPIKeyNameQuestion(KeyLocation.Header));
```

Comment:
we may need to pass localized string `plugins.apiConnector.headerLocationOption.title` here, otherwise the text displayed to user will have non localized part